### ISSUE TITLE
ffmpeg version

### ISSUE THREAD

**Issue by georgesterpu** [menpo/menpo#804]
Hi
@JeanKossaifi @patricksnape 

Why does menpo require `ffmpeg >=2.7,<3.0` ?

When I don't change the `MENPO_FFMPEG_CMD` environment variable and have` ffmpeg 3.3` installed on my system, menpo is using that one instead of 2.7 from the conda channel, and my final results are not identical for the two `ffmpeg` versions. Are the more recent builds known to cause problems in menpo ?

Thanks,
George

**Comment by patricksnape** [comment]
They are not known to cause issues - though using libav masquerading as ffmpeg on older Ubuntu installs was causing an issue. Decoding video codecs is complicated and it's very difficult to replicate results across machines! Are you seeing actual errors or just different numbers of frames?

**Comment by georgesterpu** [comment]
I did not get any error, and it is arduous to check the frame count for every video right now. With `exact_frame_count=True`, I guess everything should match.

The only obvious difference that I observed was the `result.final_error()` value, often this is from the 8th decimal, but sometimes from the first one too.

I did another check now and I should update my initial message: the system ffmpeg (`/usr/bin/ffmpeg`) is used when running the scripts from pycharm, while the local one (`miniconda3/envs/menpo/bin/ffmpeg`) is used when running from terminal the instead.

I understand that the decoded video is likely to be different across multiple versions of the same codec, was just wondering if the results with the newer versions are still correct, given the requirement in the .yml file.
